custom-commands/demos/scripts/deployCustomCommands.ps1

```diff
@@ -132,7 +131,11 @@ write-host "...training is completed"
 #
 
 write-host "publishing the model"
-$response = invoke-restmethod -Method POST -Uri "https://$CustomCommandsRegion.commands.speech.microsoft.com/apps/$appId/stages/default/cultures/en-us/publish/$versionId" -Header $headers
+# Direct following code will be interrupted by response status 405.
+# $response = invoke-restmethod -Method POST -Uri "$($OperationLocation.replace('/train/','/publish/'))" -Header $headers
+# Direct following code will not produce error, but response status 405 still is the result, which is same as running train-and-publish.sh. WVAC can not test properly after deployment.
+$response = curl -s -X POST "$($OperationLocation.replace('/train/', '/publish/'))" -H "Ocp-Apim-Subscription-Key: $speechResourceKey" -H "Content-Length: 0"
+
```

Comment:
What's the status of the above? I don't understand the comments.

Reply:
Using curl as it does in train-and-publish.sh, the publish will not generate error. But the publish actually is not done successfully, and WVAC can not connect deployed CC app. I IMed Enrique to report this issue. Please see my IM messages.

For now, I suggested still using current version and upgrade after v1.0 is formally released.

Reply:
Fixed and tested.